### ISSUE TITLE
Nether portals Spawn when teleporting cross dimension.

### ISSUE THREAD

**Issue by lilrichy** [ForgeEssentials/ForgeEssentialsMain#1447]
When ever using the /back or a /warp command to travel to a different dimension it will spawn a Nether Portal in the dimension you arrive in close by. This is a "cheaty" way to get a Nether Portal before you have the resources to make one. This also is making a lot of random Nether Portals in the world due to players using /back when they die.

**Comment by evernife** [comment]
This happens to me only when creating a MultiWord. But in fact, need a fix :D

**Comment by lilrichy** [comment]
I should probably add in that I am not using any kind of Multi World / mystcraft mod I noticed it when teleporting from the extra utilities mod "-112 dimension; The Last Millenium" where I built my server spawn, to the Over world. I have the Global spawn point in the last millenium so when someone dies they get sent back to the server spawn. Then if they use the /back command, there will be a nether portal near by in the overworld where they teleported to. The nether portals where showing up in my spawn area when someone would use /warp spawn so I just hid one out of sight and that seems to stop the random spawning of them there but this is impossible to do in the over world.

**Comment by olee** [comment]
Interesting bug.
Could you write a little step by step guide to reproduce the issue?

**Comment by lilrichy** [comment]
This is a link to my mod list idk if it will help or not http://pastebin.com/R5fD5bd6

Make warp called spawn in dim -112
Make warp called overworld in dim 0
Use /warp overworld while in dim -112 will teleport and spawn nether portal
Use /warp spawn while in dim 0 will teleport and spawn nether portal
Using /back also does same thing if going cross dimension

Nether portals do not appear if /warp or /back in Same dimension only when teleporting cross dimension. 

Also using /cofh tpx will teleport cross dimension without making a nether portal. Seems to only happen when using ForgeEssentials teleport commands.

**Comment by olee** [comment]
Thanks. We will investigate and fix this ASAP.

**Comment by tethlah** [comment]
I've also noticed that any time you use /home or /spawn it creates a random nether portal somewhere

**Comment by olee** [comment]
Sorry, somehow forgot to assign this issue to me and lost a bit track of it.

**Comment by olee** [comment]
I was not able to reproduce this bug. Can someone give me a working example on how to reproduce this?

**Comment by tethlah** [comment]
Use any FE teleport command to teleport to overworld from any dimension other than overworld.  It's completely random (at least I haven't seen a pattern or reason as to why or where) where the portal will spawn.  But it's definitely from FE because that's the only mod added to my server, and the portals start cropping up all over the place that day.

**Comment by skittishtrigger** [comment]
Confirming this issue. I thought my players were being lazy and building random portals.  up on further investigation it happens exactly as tethlah has stated earlier.

**Comment by ShicKla** [comment]
After some researching I found that calling the vanilla class `transferPlayerToDimension()` to teleport across dimensions can/will cause this issue, but does not occur 100% of the time.

Going through FE's source I found that you are using this call in `main/java/com/forgeessentials/core/misc/TeleportHelper.java`

Two possible solutions to this problem is to create/use your own custom code to handle cross dimension teleportation, or give/have the ability to disable auto generation of nether portals. Unfortunately both of these solutions are easier said then done.
